library/src/factory.cxx.in

```diff
@@ -62,7 +62,7 @@ void factory::load(plugin * r)
 {
   if (!this->registerOnce(r))
   {
-    log::warn("Plugin \"" + r->getName() + "\" is already loaded.");
+    log::debug("Plugin \"" + r->getName() + "\" is already loaded.");
```

Comment:
log ?

Reply:
should be displayed by default? (without `--verbose`)

Reply:
my bad, you are right